### ISSUE TITLE
Blurry screenshots

### ISSUE THREAD

**Issue by adelnoureddine** [markathomas/vaadin-screenshot#3]
Hi, are there options for quality/resolutions for the screenshots taken?
The screenshots I am taking are blurry. The component in question is a gridLayout filled with coloured verticalLayouts.

![download](https://cloud.githubusercontent.com/assets/1515666/20133642/4891e98e-a660-11e6-9bd8-2d424e5294d1.png)

**Comment by markathomas** [comment]
no, there are no options for html2image.js that allow resolutions as far as
i know

Regards,

Mark Thomas
spatialguru.net@gmail.com
205.529.9013

"Commit to the Lord whatever you do,
    and your plans will succeed." - Proverbs 16:3

**Comment by adelnoureddine** [comment]
Thanks Mark for the information.
